app/components/earn/EarnHeader.tsx

```
@@ -0,0 +1,110 @@
+}
+
+export function EarnHeader({ stats, loading }: EarnHeaderProps) {
+  return (
+    <div className="relative">
+      {/* Background grid fade */}
+      <div className="absolute inset-x-0 top-0 h-48 bg-grid pointer-events-none" />
+
+      <div className="relative mx-auto max-w-6xl px-4 pt-10 pb-6">
+        {/* Section tag */}
+        <div className="mb-2 text-[10px] font-medium uppercase tracking-[0.25em] text-[var(--accent)]/60">
+          // earn
```

Biome (2.4.4)

[error] Wrap comments inside children within braces.

(lint/suspicious/noCommentText)